src/runtime_strings_extra.js

```diff
@@ -62,16 +62,16 @@ function UTF16ToString(ptr, maxBytesToRead) {
   } else {
 #endif // TEXTDECODER != 2
 #endif // TEXTDECODER
-    var i = 0;
-
     var str = '';
-    while (1) {
+
+    for (var i = 0; !(i >= maxBytesToRead / 2); ++i) {
```

Comment:
Would this not be more readable as `i < (maxBytesToRead / 2)`?

Reply:
Definitely :) But it's done this way so that is also works if `maxBytesToRead` is `undefined`. In this case the condition is always true and the loop is terminated on the first null char. Since the same is done in a couple of other places (a few lines above and in other string conversion functions), I would like to keep it like this, but will add a comment.

Reply:
Perfect, thanks!